docs/src/introduction/get-help-get-involved.md

```diff
@@ -0,0 +1,31 @@
+# Get Help and Get Involved
+
+To support the open-source community, we provide public-facing Zeebe support channels and a way for users to report problems and make contributions.
```

Comment:
I would remove the second support (`Zeebe support channels`) to avoid repeating it

Reply:
Agree, updated.